### ISSUE TITLE
v2.0 Release Candidate Testing

### ISSUE THREAD

**Issue by CaMer0n** [e107inc/e107#1174]
Please test and post feedback. @tgtje @nlstart @MikeyGMT @Moc @myovchev 

http://e107.org/_misc/e107_2.0_full.zip

http://e107.org/_misc/e107_v1.x_to_2.0_upgrade.zip

If no serious issues found, these will be moved to sourceforge and become the official v2.0 release.

**Comment by tgtje** [comment]
First overall impression on v2 FULL > no issues when installing.
I only had a visual impression on ALL (for any misbehaviour and or missing things) 
I did not use it throughout, only the parts i think need to be checked. 
As said : impression ( i trust the system enough to go ahead for now).
testconfig:
using Xampp :
Apache/2.4.10 (Win32) OpenSSL/1.0.1i
(host: localhost)
PHP Version
5.6.3
MySQL
5.6.21
-------------------------------
Visual error : 
ADLAN_24 in search configuration 
search internal (on files) > define (ADLAN_24) is DEcommented (English/admin/lan_admin php)
probably due to recent changes on search ????
 -----
file inspector reports 1 missing file = install php (correct)
------
used Stop Forum Spam for download (from within system) >> download unpack = succes
(refreshment pluginmanager) 

Message : ! 
Success
'' is missing from the plugin db table

stop forum spam is found in uninstalled plugins, install > ok
-------
theme manager ( through online, used Theme basic for download)
Theme is present in change theme (install works)

(while at it, used the external upload function > AtiSilver (old one)
uploading.unzip/change theme/install > ok

---- no complaints----

Next step : upgrade ( have to investigate> step 6 for forum does not adequate run> reason  ??)
can not post more before more info is present (maybe related to the multiple languages ??)

**Comment by CaMer0n** [comment]
Thank you @tgtje . The .zip files above have been updated with some changes as referenced above. 
So, if I understand well, there is still an issue with the Forum upgrade?

**Comment by tgtje** [comment]
@CaMer0n  yes and no. ellaborate: i had an issue with 1 dbase and it stopped at step 6 for forum.
resetting will ultimately disturb routines since tables are already present ( is not expected ( understanding of what reset means), but kind of normal behaviour. step 4 etc can not create them again.. ( so manual sql deletion and you can start over).
On that specific dbase i just suspect theres more in the dbase itself , rather than the upgrade process.
It used three languages, had some issues of 'half' unistalled plugins (but dbase contained leftovers.etc.
And because of types of plugins used (past eg during 2014) had all kinds of collation (swedish, innodb,utf 8)..Forum itself had (due to language use, not all similar table 'content' : sometimes dutch had 25 tables, english 32 etc..) all that leads me to the assumption > dbase related. ( in todays standards i would say > oh my G... how could it have worked ....hihi, but it did once..)

Restarting the process again with the dbase i used in earlier tests ( again all is 'fresh' loaded (dbase and files belonging to each other) i have NO troubles..( and alike the earler pre-tests for other issues) it runs fine.
Is it possible to pinpoint why 1 dbase can and the other doesn't ; imo NO (it would involve in a comlete dbase rewrite (and in such doing i will surely 'repair'the issue so the outcome would be useless for testing).

I will do a retest again after i have got a old of other dbases and files (from clients), so i can investigate even further. (have to find one that uses a lot of plugins incl. forum)
In that case i will do run again using the newer files.

Best case scenario would be : more testing of different people and their results, else all is (would be) to much 'coloured', since i use the system not as a 'basic' user (allthough i try to act like so when testing : eye of the beholder).

**Comment by tgtje** [comment]
Well some hours later, did obtain 4 dbases + files
2 without forum plugin in use > upgrade > ok
e107adminlogo not displayed >paths on bootstrap, adding bootstrap theme > its there

1 with forum ( a small one forum injected) > ok (but ONLY after i did delete a used plugin older e107slider..)

1 with forum (larger one) > can not proceed further than step 6 (freezes), debugger does not display aything at that point) tried a lot of possibilities.. still no go (upgrading files on this one also eliminates admin top sitelinks) ( 2 collation types (swedish + utf8 ) unable to pinpoint what does where wrong, in FF getting referenceerrors on jquery (core) so in regards to he dbase above> will a jquery error or conflict mess up ???................. :(

Assuming now ; forum update is a pain.... ( to much variables to be taken in consideration..)

**Comment by CaMer0n** [comment]
@tgtje Can you email me some login information so I can have a look at this issue?

**Comment by tgtje** [comment]
Stored clientdata on Xampp, will see if a can setup a subdomain with full acces (ftp and db) on live server.
Can not provide access to original real live site.

**Comment by tgtje** [comment]
Mail send, hope it arrives..

**Comment by Grimston** [comment]
Works well so far.

### Forum
User profile links in the forum posts are incorrect when using friendly links (Have not tested normal links)
```
http://e107.localhost/forum/general/1-test/user.php?id.1
```
So it does nothing but reload the current page.

Also Admin > Forum > Manage, Forums are not under the parents, but are listed in the order created.

Example
![e107v2forummanage](https://cloud.githubusercontent.com/assets/4490380/9631572/6ecbfcf4-51b4-11e5-9349-0937f7f537be.jpg)


### Login/Logout
Upon logout, the session is still in the database, which stops the user from logging in again, when disallowing multi logon. Note: A NEW session was created at logout, so old ones are not being removed.

**Comment by CaMer0n** [comment]
@tgtje Nothing received. Please use the contact form at e107.org. thanks.

**Comment by CaMer0n** [comment]
Closing. Please create a new issue if needed.
